docs/snippets/proc-subscribe-function.md

```diff
@@ -31,14 +31,3 @@ and a Knative Broker as the source of events, from where they are consumed.
     ```bash
     kn func subscribe --filter type=com.example --filter extension=my-extension-value
     ```
-
-!!! note
-    When invoking `func deploy` the CLI will create Knative Triggers for the function.
-
```

Comment:
@matzew have you intentionally removed the note box?

Reply:
yes, I have. please see the rendered page